content/en/developers/dogstatsd/_index.md

```diff
@@ -18,6 +18,9 @@ further_reading:
     - link: 'https://github.com/DataDog/datadog-agent/tree/main/pkg/dogstatsd'
       tag: 'GitHub'
       text: 'DogStatsD source code'
+    - link: "https://www.datadoghq.com/blog/monitor-azure-app-service-linux/"
```

Comment:
Just FYI, I think you might be missing the Further reading partial.

Reply:
@drichards-87 oh yeah, that was never there. Just added, thanks!!